_episodes/06-style-guide.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Style Guide"
+teaching: 10
+exercises: 0
+questions:
+- "How are keyboard key combination write?"
```

Comment:
Written instead of write, I think.

Reply:
@rgaiacs, Perhaps adjust grammar to:
`How are keyboard key combinations written? or How is a keyboard key combination written?`